programming basics/Conda and Git.ipynb

```diff
@@ -0,0 +1,134 @@
+{
```

Comment:
<p>Maybe a cell with download link to <code>conda</code> can be added</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/PTC-CMC/cybercamp/pull/11/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>